### ISSUE TITLE
preparing images for dense reconstruction failed (errors loading images)

### ISSUE THREAD

**Issue by SherlockCheung** [cdcseacave/openMVS#950]
I wanted to change the DensifyPointCloud.cpp content, but I entered my scene.mvs in OPT::strInputFileName and CMD reported an error with the following content:

18:25:56 [App     ] error: failed loading image header
18:25:56 [App     ] error: failed loading image header
18:25:56 [App     ] error: failed reloading image 'C:/Users/lenovo/Desktop/openmv/openMVS-master/build/apps/DensifyPointCloud/undistorted_images/IMG_20221010_145115.jpg'
18:25:56 [App     ] error: failed reloading image 'C:/Users/lenovo/Desktop/openmv/openMVS-master/build/apps/DensifyPointCloud/undistorted_images/IMG_20221010_145124.jpg'
18:25:56 [App     ] error: preparing images for dense reconstruction failed (errors loading images)

but the images are available when I used the CMD command: "./DensifyPointCloud scene.mvs".
So my issue is how do I modify the code (my input path) to make it work.Thank you.

**Comment by cdcseacave** [comment]
most probably there is an error in building the project, you mixed different versions of JPG library, it happened before, follow the build instructions

**Comment by SherlockCheung** [comment]
Thank you for your answers.I had used the vcpkg.exe to build the project which also follow the build instructions,but it won't work.I discovered that  CMakeList file(OpenMVS) used opencv which vcpkg

**Comment by cdcseacave** [comment]
no, if everything is built by VCPKG it will work, see the test build steps in https://github.com/cdcseacave/openMVS/blob/master/.github/workflows/continuous_integration.yml

**Comment by oferkITG** [comment]
I have the same problem, 
any ideas how to resolve?

**Comment by cdcseacave** [comment]
This error occurs when image files cannot be found at the paths recorded in the \.mvs\ file. Verify image paths are correct and accessible from the working directory.
